content/learning-paths/servers-and-cloud-computing/cca-container/hello-realm.md

```diff
@@ -0,0 +1,136 @@
+---
+# User change
+title: "Run an application in a Realm"
+
+weight: 3 # 1 is first, 2 is second, etc.
+
+# Do not modify these elements
+layout: "learningpathall"
+---
+
+## Before you begin
+
+Make sure you have downloaded the [`armswdev/aemfvp-cca-image`](https://hub.docker.com/r/armswdev/aemfvp-cca-image) docker container.
```

Comment:
"download the <> docker container image" is probably more correct :-)

Reply:
Agreed, corrected :)